### DISCUSSION TITLE
Index/show images stored on NAS

### DISCUSSION THREAD

**Discussion by tomasenskede** [photoprism/photoprism#4477] [Q&A] [answered]
How can I configure Photoprism to index all my pictures stored on my NAS without moving them?

I've mapped the NAS share to the Photoprism server under /path/to/pictures, and I can access the files. What should I do next?

In the import dialog within Photoprism, I don't see an option to specify the /path/to/pictures share for import...

**Comment by tomasenskede** [comment]
should I add the following lines in options.yml?

import_path: /path/to/pictures
originals_path: /path/to/pictures

Aug 24 11:38:26 photoprism photoprism[518]: time="2024-08-24T11:38:26+02:00" **level=warning msg="config: failed loading values from /opt/photoprism/storage/config/options.yml** (yaml: unmarshal erro>

Aug 24 11:38:26 photoprism photoprism[518]: time="2024-08-24T11:38:26+02:00" level=info msg="daemon started with process id 525\n"
Aug 24 11:38:26 photoprism systemd[1]: Started photoprism.service - PhotoPrism service.
lines 184-223/223 (END)

**Reply by tomasenskede** [reply]
Sems like the line "ImportPath: /path/to/pictures" should be added in the options.yml only...

**Reply by srett** [reply]
The import path is from where PhotoPrism should move new files to the originals folder, either automatically or when you hit import. In your case you should set the originals path to your NAS path, since you don't want to have PP do any importing of files, but just use it to index your existing collection.

**Reply by tomasenskede** [reply, marked as answer]
import_path: /path/to/pictures

Was the config needed. Now it can import images from my NAS
